core/app/models/spree/payment.rb

```diff
@@ -206,6 +210,20 @@ def invalidate_old_payments
         end
       end
 
+      def split_uncaptured_amount
+        if uncaptured_amount > 0
+          order.payments.create amount: uncaptured_amount,
```

Comment:
Probably want this to either be create! or to verify in some other way that it was created.